tests/func/test_config.py

```diff
@@ -278,3 +284,13 @@ def test_config_show_origin(tmp_dir, dvc, caplog):
         )
         in caplog.text
     )
+
+    caplog.clear()
+    assert main(["config", "--list", "--show-origin"]) == 0
+    assert (
+        "{}\t{}\n".format(
+            os.path.join(".dvc", "config"),
+            "remote.myremote.url=s3://bucket/path",
+        )
```

Comment:
We need a test for multiple levels in one output, just testing .dvc/config is not enough, we need to at least try it + .dvc/config.local
```suggestion
    assert (
        "{}\t{}\n".format(
            os.path.join(".dvc", "config"),
            "remote.myremote.url=s3://bucket/path",
        )
```

Also, this `test_` is pretty packed now, please consider creating separate test cases for new tests.

Reply:
Done.